### PR TITLE
deleted SKU and IMB form the node table

### PR DESCRIPTION
**Pull request by danieltolmon**
enhancement #7 

I deleted the columns of the SKU and IMB from the node table that were not necessary.

It creates more changes because I used prettier. Which  format source code do you use?

### CODE REVIEW

**Comment by danieltolmon** [comment]
I could also hide them as default. And then, if the user wants, he could select them to be shown

**Comment by jsaguiar** [comment]
That would be nice

File: src/app/management-center/nodes/nodes.component.html [hunk 1]
```diff
@@ -144,8 +132,7 @@ <h3 class="modal-title">Create Node</h3>
     </div>
   </clr-modal>
 
-  <confirm-modal [(isPop)]="isDelete" title="nodes"
-    (confirm)="onConfirm($event)" [data]="selectedNodes">
+  <confirm-modal [(isPop)]="isDelete" title="nodes" (confirm)="onConfirm($event)" [data]="selectedNodes">
   </confirm-modal>
 
-</div>
+</div>
```
Comment (jsaguiar) [inline src/app/management-center/nodes/nodes.component.html]: add new line

**Comment by danielgek** [comment]
LGTM apart from Aguiar comment

**Review by asantos00 — APPROVED** [review]
👍

**Comment by danieltolmon** [comment]
I changed the columns to be hidden by default instead of being deleted. So the user can have them, in case he wants.

**Comment by asantos00** [comment]
Have you pushed those changes? @danieltolmon

**Comment by danieltolmon** [comment]
Yes, it 's the last commit: [
8499f78](https://github.com/KI-labs/on-web-ui/pull/14/commits/8499f787dc7c26a81b31c271ff499ae310bf609d)

**Comment by danieltolmon** [comment]
@danielgek, @jsaguiar, @aramoski 
Guys I need some feedback!! Do I merge this PR? What about the others? :)

**Comment by danielgek** [comment]
This one looks good to me,  @jsaguiar

**Pull request merged**